_inc/client/at-a-glance/activity.jsx

```diff
@@ -48,8 +51,57 @@ class DashActivity extends Component {
 		// 	} );
 
 		// @todo: update this to use rewind text/CTA when available
-		const activityLogOnlyText = __(
-			'Jetpack keeps a complete record of everything that happens on your site, taking the guesswork out of site management, debugging, and repair.'
+		const months = [
+			__( 'Jan' ),
+			__( 'Feb' ),
+			__( 'Mar' ),
+			__( 'Apr' ),
+			__( 'May' ),
+			__( 'Jun' ),
+			__( 'Jul' ),
+			__( 'Aug' ),
+			__( 'Sep' ),
+			__( 'Oct' ),
+			__( 'Nov' ),
+			__( 'Dec' ),
+		];
+		const activityLogOnlyText = isEmpty( siteActivity ) ? (
+			<p>
+				{ __(
+					'Jetpack keeps a complete record of everything that happens on your site, taking the guesswork out of site management, debugging, and repair.'
+				) }
+			</p>
+		) : (
+			<ul className="jp-dash-activity__list">
+				{ siteActivity.map(
+					( { summary, content: { text }, actor: { name, icon }, published }, index ) => {
+						const dateTime = published.split( 'T' );
+						dateTime[ 0 ] = dateTime[ 0 ].split( '-' );
+						return (
+							<li key={ `activity-event-${ index }` }>
+								<time dateTime={ published } className="jp-dash-activity__date-time">
+									<div className="jp-dash-activity__date">{ `${
+										months[ parseInt( dateTime[ 0 ][ 1 ] ) ]
```

Comment:
I think this may prove problematic in foreign languages, where the date format may be different. Maybe we should make the whole date string translatable?

Reply:
Thanks. This is now fixed using moment.js.